model/host/db.go

```diff
@@ -1741,10 +1741,12 @@ func SyncPermanentExemptions(ctx context.Context, permanentlyExempt []string) er
 			},
 		})
 		catcher.Wrap(err, "marking newly-added hosts as permanently exempt")
-		grip.InfoWhen(res.ModifiedCount > 0, message.Fields{
-			"message":   "marked newly-added hosts as permanently exempt",
-			"num_hosts": res.ModifiedCount,
-		})
+		if res != nil && res.ModifiedCount > 0 {
```

Comment:
We should probably also add it to line 1703 and 1744

Reply:
L1703 won't panic because it already checks if the err is non-nil.

L1744 is this line?

Reply:
> L1744 is this line?

yes lol, my bad. I grepped too quickly. 

you're right, it looks like res is only nil when the err is non-nil so we should be good!